### PR TITLE
WebUI stash list now shows diff for selected stash item

### LINKED ISSUE #265: WebGUI: the differance for stashed changes not displayed
When using the WebGUI to manage stashed changes in the local (server side) GIT repository the UI does not update the selected left hand stashed change but always shows the latest stashed item a the time the WebGUI did load. Plz refer to the screenshot.  
(switching commits in a branch works just fine).

image

### PR DESCRIPTION
**Pull request by isc-hwojnick**
Fixes #265

### CODE REVIEW

File: git-webui/release/share/git-webui/webui/js/git-webui.js [hunk 1]
```diff
@@ -1256,11 +1256,11 @@ webui.StashCommitView = function(stashView) {
     var self = this;
 
     self.update = function(entry) {
-        if (currentCommit == entry.commit) {
-            // We already display the right data. No need to update.
-            return;
+        // if (currentCommit == entry.commit) {
```
Comment (isc-tleavitt) [inline git-webui/release/share/git-webui/webui/js/git-webui.js:1259]: Remove dead code, don't just comment it out!

**Review by isc-tleavitt — CHANGES_REQUESTED** [review]
Please clean up the code and add CHANGELOG.md. This also picked up changes from another PR which isn't the end of the world.

**Comment by isc-hwojnick** [comment]
> Please clean up the code and add CHANGELOG.md. This also picked up changes from another PR which isn't the end of the world.

Ah I must have accidentally made this branch from another branch to fix a different issue instead of from main

File: git-webui/release/share/git-webui/webui/js/git-webui.js (hunk 1 above)
Comment (isc-tleavitt) [reply git-webui/release/share/git-webui/webui/js/git-webui.js:1259]: Still relevant

Comment (isc-hwojnick) [reply git-webui/release/share/git-webui/webui/js/git-webui.js:1259]: This is in the release file. I just didn't recompile after removing the dead code. Is that fine?

**Review by isc-tleavitt — APPROVED** [review]

**Pull request merged**